cli/file_watcher.rs

```diff
@@ -204,7 +209,8 @@ where
     let _watcher = new_watcher(&paths, &debounce)?;
 
     if let Some(module) = &module {
-      let func = error_handler(operation(module.clone()));
+      let func =
+        error_handler(operation(pstate.clone().unwrap(), module.clone()));
```

Comment:
this is funky... why do we wrap it in an option to only clone it an unwrap it‽

Reply:
Indeed, this is because of how this function is structured. I'll try to rework it in a follow up